frontend/js/components/TableActiveRow.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+import pokeball from '../../image/pokeball.svg';
+
+const StyledRow = styled(Link)`
+  background-color: #fff;
+  padding: 1rem 2rem;
+  border-radius: 30px;
+  display: grid;
+  grid-template-columns: 40px repeat(4, 1fr);
+  box-shadow: 0px 0px 15px -12px rgba(0, 0, 0, 0.74902);
+  text-align: center;
+  font-size: 0.8rem;
+  margin-bottom: 1rem;
+`;
+
+const StyledImage = styled.img`
+  width: 20px;
+`;
+
+const StyledCall = styled.span`
+  position: relative;
+  &&:after {
+    content: 'Answer';
+    color: #fff;
+    background-color: #f9ab2f;
+    padding: 5px;
+    position: absolute;
+    right: -40px;
+    bottom: -5px;
+    font-weight: 500;
+    outline: dashed #29344c;
+  }
+`;
+
+const getLink = (opponent, user, id) => {
+  if (user === opponent) {
+    return `/create-team/${id}`;
+  }
+  return `/battle/${id}`;
+};
+
+const TableActiveRow = (props) => {
```

Comment:
**Small tip:**

You can do this
```js
const { id: battleId, created, user_opponent, user_creator } = battle;
```
Instead of using the whole object path. This can be applied to any object, and makes things easier to read!